pkg/server/singleprocess/service.go

```diff
@@ -36,6 +36,11 @@ type Service struct {
 	// with EncodeId), and returns only the waypoint-relevant ID.
 	decodeId func(encodedId string) (id string, err error)
 
+	// processToken allows an implementation the ability to alter a to be
+	// generated token before it's generated. This can be used to alter semantics
+	// about the token, such as adding labels, metadata, or additional info
```

Comment:
Looks like a typo here?
```suggestion
	// processToken allows an implementation the ability to alter a token to be
```
I'm also not sure what it means to alter a token to be a generated token before it's generated? Like structurally?

Reply:
Yeah, as in right before the token is returned for something to use, processToken is handed the token and can alter it.